### PR TITLE
Remove page descriptions where they are not needed #550

### PR DESCRIPTION
**Pull request by SebinSong**
Resolving issue #550 in 'Dashboard' project.

- got rid of the corresponding html elements from each Vue component.

### CODE REVIEW

**Comment by SebinSong** [comment]
I'm fixing a bunch of linter errors, regarding travis-ci failing and one of them is

linter-error


and this is my code change for that error.

fixed-linter-error

would it be safe and acceptable? :)

The original code was,
let matches
while (matches = functionEx.exec(text)) {
......
}

File: frontend/views/GroupDashboard.vue [hunk 1]
```diff
@@ -8,9 +8,6 @@
       <h1 class="title is-1 is-marginless" data-test="groupName">
         {{ currentGroupState.groupName }}
       </h1>
-      <p data-test="sharedValues">
-        {{ currentGroupState.sharedValues }}
-      </p>
```
Comment (taoeffect) [inline frontend/views/GroupDashboard.vue:13]: Looks like removing this breaks the tests.

Here are my thoughts in general on this PR, and I could also use some help from @pieer and @mmbotelho on this: should this issue maybe be incorporated as part of a larger PR to implement the radically changed UI that @mmbotelho designed on Figma? Or should it be merged as is with fixed tests? What do you guys think?

Comment (mmbotelho) [reply frontend/views/GroupDashboard.vue:13]: I would say that merging is better than waiting for a massive PR to be done. I created these issues with that in mind - doing small pieces of code and merging them as fast as possible. @pieer what do you think?

Comment (pieer) [reply frontend/views/GroupDashboard.vue:13]: @taoeffect Yes fix the test and merge, I will update the full page soon anyway
@mmbotelho Breaking issues into small tasks is better indeed

File: frontend/views/containers/NavBar.vue [hunk 2]
```diff
@@ -91,7 +91,6 @@
         </div>
       </div>
       <time-travel v-show="ephemeral.showTimeTravel" :toggleVisibility="toggleTimeTravel" />
-    </div>
```
Comment (taoeffect) [inline frontend/views/containers/NavBar.vue:94]: Nice catch! And yeah in the future you are also welcome to fix the indentation on the file by re-indenting the entire block left (as it should be). :-)

**Review by taoeffect — APPROVED** [review]
Great job @SebinSong!

File: frontend/views/containers/NavBar.vue (hunk 2 above)
Comment (SebinSong) [reply frontend/views/containers/NavBar.vue:94]: Ok! 👍

**Pull request merged**